src/odemis/gui/comp/buttons.py

```diff
@@ -845,9 +852,9 @@ def set_overlay_image(self, image):
             scaled_img = wxImageScaleKeepRatio(image, self.thumbnail_size, wx.IMAGE_QUALITY_HIGH)
         else:
             # black image
-            scaled_img = wx.EmptyImage(*self.thumbnail_size)
+            scaled_img = wx.Bitmap(*self.thumbnail_size)
```

Comment:
This should be an _image_ not a bitmap. With GTK3, this caused very noticeable issues, the simple fix is to write:
`scaled_img = wx.Image(*self.thumbnail_size, clear=True)`

Reply:
Clear